recipes/scanning-drift-corr/run_test.py

```diff
@@ -0,0 +1,17 @@
+import numpy as np
+import scanning_drift_corr.api as sdc
+
+scanAngles = (10, 100)
+
+n = 9
+im1 = np.ones((n,n))
+im2 = np.ones((n,n))
+im1[4,6] = 10
+im1[2,3] = 10
+im2[5,2] = 10
+im2[2,4] = 10
+
+sm = sdc.SPmerge01linear(scanAngles, im1, im2)
+sm = sdc.SPmerge02(sm, 8, 8)
+imageFinal, signalArray, densityArray = sdc.SPmerge03(sm)
+
```

Comment:
It looks like it gets stuck in an infinite loop:
```
RuntimeError: 
        An attempt has been made to start a new process before the
        current process has finished its bootstrapping phase.

        This probably means that you are not using fork to start your
        child processes and you have forgotten to use the proper idiom
        in the main module:

            if __name__ == '__main__':
                freeze_support()
                ...

        The "freeze_support()" line can be omitted if the program
        is not going to be frozen to produce an executable.
```

Reply:
Now it's clearer that the package is actually broken on macOS and Windows. Can you open an issue upstream?

As it's `noarch: python` the issue can be ignored for conda-forge as it'll only be built on Linux after this is merged.